app/src/main/java/ch/epfl/sweng/swenggolf/statistics/StatisticsActivity.java

```diff
@@ -0,0 +1,90 @@
+package ch.epfl.sweng.swenggolf.statistics;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+
+import ch.epfl.sweng.swenggolf.R;
+import ch.epfl.sweng.swenggolf.database.DbError;
+import ch.epfl.sweng.swenggolf.database.ValueListener;
+import ch.epfl.sweng.swenggolf.profile.User;
+import ch.epfl.sweng.swenggolf.tools.FragmentConverter;
+
+public class StatisticsActivity extends FragmentConverter {
+    private View inflated;
+    private User user;
+    private StatisticsAdapter statisticsAdapter;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        final Bundle bundle = getArguments();
+        user = bundle.getParcelable(User.USER);
+    }
+
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstance) {
+        setToolbar(R.drawable.ic_baseline_arrow_back_24px, "Statistics");
+        inflated = inflater.inflate(R.layout.activity_statistics, container, false);
+        getStats();
+        return inflated;
+    }
+
+    @Override
+    public void onDestroyView() {
+        super.onDestroyView();
```

Comment:
I think that if you just call the superclass method here, you might not need to overwrite it.

Reply:
Indeed!